backend/src/index.d.ts

```diff
@@ -0,0 +1,9 @@
+import 'fastify';
+
+import { type UserSignUpResponseDto } from '#packages/users/users.ts';
```

Comment:
Is this extension in correct form?

Reply:
do you recommend using schema from here https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/issues/12#issuecomment-1687974980 ?

Reply:
No, I mean about .ts extension)

Reply:
oh, I see) Will change it